.changeset/afraid-ducks-listen.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/presentation-hierarchies-react": patch
```

Comment:
Should it not be a minor version bump because a new property was added to `TreeNodeRenderer`?

Reply:
Added change file for new prop.